src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -0,0 +1,123 @@
+package com.github.manolo8.darkbot.utils;
+
+import com.github.manolo8.darkbot.gui.utils.Popups;
+
+import javax.swing.*;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class StartupParams {
+    private static final String COMMAND_PREFIX = "-";
+
+    /**
+     * <b>NAME</b>
+     * <pre>
+     *     login - auto logins to the account with this <i>username</i>
+     * </pre>
+     * <b>SYNTAX</b>
+     * <pre>
+     *     -login <i>username</i> <i>filepath</i>
+     * </pre>
+     * <b>OPTIONS</b>
+     * <pre>
+     *     <i>username</i> and <i>filepath</i> are required arguments
+     *     <i>username</i> refers to the account you want to login to
+     *     <i>filepath</i> refers to the filepath to your txt file containing your master password
+     * </pre>
+     * <b>SAMPLE USAGE</b>
+     * <pre>
+     *     -login NotABot C:\Users\Owner\masterpw.txt
+     * </pre>
+     * @see #masterPasswordFile
+     */
```

Comment:
```suggestion
    /**
     * Command-line argument for auto-login, requires username and a path to a plaintext file with the master password.
     * Example usage: {@code -login NotABot C:\Users\Owner\masterpw.txt}
     * @see #username
     * @see #masterPasswordFile
     */
```
It feels like you're writing documentation for end-users, but this is documentation for developers, keep it simple